modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,63 @@
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Control plane::
+Th control plane manages the worker nodes and the pods in the {product-title} cluster. It is a container orchestration layer that exposes the API and interfaces to define,deploy, and manage the lifecycle of containers.
+//re-written defintion
+
+Installation program::
+The installation program is a program that installs certain software. It is a Go binary file that performs a series of file transformations on a set of assets.
+//re-written and also taken from existing doc
+
+Installer-provisioned infrastructure::
+The installation program deploys and configures the infrastructure that the cluster runs on.
+// re-written from ocp repo glossary
+
+User-provisioned infrastructure::
+The user deploys and configures separate virtual or physical hosts as part of the cluster deployment process.
+// re-written from ocp repo glossary
+
+Compute node::
+Compute nodes also known as worker nodes, are nodes which are responsible for executing workloads for the cluster users.
+//re-written
+Bootstrap node::
+A bootstrap node is a temporary machine that runs a minimal Kubernetes configuration that is used to deploy the {product-title} control plane. It is deleted at the end of the installation.
+//re-written
+
+Machine Config Operator::
+The Machine Config Operator manages and applies configuration and updates of the base operating system and container runtime, including everything between the kernel and kubelet.
+//from OCP docs
+
+Ignition config files::
+The file that Ignition uses to configure Red Hat Enterprise Linux CoreOS (RHCOS) during operating system initialization. The installation program generates different Ignition config files to initialize bootstrap, control plane, and worker nodes.
+//RH docs
+
+Kubernetes manifests::
+Kubernetes manifests are the specifications of a Kubernetes API object in a JSON or YAML format. A configuration file can include deployments, config maps, secrets, daemonsets etc.
+//re-written
+
+Load balancers::
+A load balancer service allocates a unique IP from a configured pool. The load balancer has a single edge router IP, which can be a virtual IP (VIP), but is still a single machine for initial load balancing).
+//ocp docs
+
+Kubelet::
+A **primary node** agent that runs on each node in the cluster. It ensures that containers are running in a pod.
+//re-written
+
+Operators::
+Operators are the preferred method of packaging, deploying, and managing services on the control plane.
+//OCP docs
+Disconnected installation::
+There are situations where parts of a data center might not have access to the internet, even through proxy servers. You can still install {product-title} in these environments, but you must download the required software and images and make them available to the disconnected environment.
```

Comment:
Better to use `restricted network installation` to align with our current installation guide, such as:
https://docs.openshift.com/container-platform/4.10/installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.html

Reply:
I would suggest disconnected  e.g. https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-creating-registry.html. There is an open PR to standardize on the terminology usage >> https://issues.redhat.com/browse/OSDOCS-3685